src/Faker/Provider/tk_TM/Person.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Faker\Provider\tk_TM;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @var array Turkmen person name formats.
+     */
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}a',
+    );
+
+    /**
+     * @var array Turkmen first names.
+     */
+    protected static $firstNameMale = array(
```

Comment:
Please look for a source, because is the data only comes from your head, other devs might add what's in their own heads and we can't keep a stable list.